src/firefly/js/core/background/BackgroundCntlr.js

```diff
@@ -166,17 +173,27 @@ function bgSetEmail(action) {
         dispatch(action);
     };
 }
-
 function bgPackage(action) {
     return (dispatch) => {
-        const {dlRequest, searchRequest, selectionInfo} = action.payload;
+        const {dlRequest, searchRequest, selectionInfo, isWs, params} = action.payload;
```

Comment:
Why add `isWs` and `params` separately when `params` is only used when isWs is true?  Why not use an object and just call it `wspaceInfo` or something.